webpage/webstreaming.py

```diff
@@ -0,0 +1,143 @@
+            # print("ERROR: ",form.errors)
+            # if form.validate_on_submit():
+            spectro_scope_config = {}
+            spectro_scope_config['integration_time']     = form.integration_time.data
+            spectro_scope_config['integration_factor']   = form.integration_factor.data
+            spectro_scope_config['threshold']            = form.threshold.data
+            # print("INTEGRATION TIME",form.integration_time.data)
+            # print("INTEGRATION FACTOR",form.integration_factor.data)
+            # print("THRESHOLD",form.threshold.data)
+
+            set_spectro_scope(app,**spectro_scope_config)
+
+            return redirect(url_for('set_config_spectrometer'))
+
+        else:
+            #print("INTEGRATION TIME",form.integration_time.data)
+            #print("INTEGRATION FACTOR",form.integration_factor.data)
+            #print("THRESHOLD",form.threshold.data)
+            form.integration_time.render_kw     = {'value':get_spectro_scope('integration_time',app)}
+            form.integration_factor.render_kw   = {'value':get_spectro_scope('integration_factor',app)}
+            form.threshold.render_kw            = {'value':get_spectro_scope('threshold',app)}
+
+            form.integration_time.label         = 'INTEGRATION TIME:'
+            form.integration_factor.label       = 'INTEGRATION FACTOR:'
+            form.threshold.label                = 'THRESHOLD:'
+
+        integration_time                    = get_spectro_scope('integration_time',app)*1000 #Pass integration time to ms
+        det.integration_time    = get_spectro_scope('integration_time',app)
+        det.integration_factor  = get_spectro_scope('integration_factor',app)
+        det.threshold           = get_spectro_scope('threshold',app)
+        return render_template("spectroscope.html",data_x=det.get_wavelengths(),integration_time=integration_time,form=form, auto_en= (True if "automatic" == det.operation_mode else False))
+        #return render_template("spectroscope.html",data_x=det.get_wavelengths(),integration_time=int(det.integration_time*1000),form=form)
+
+    @app.route("/default",methods=['GET','POST'])
+    def set_default_config():
+        if request.method == 'POST':
+            # with lock:
+            #     delete_db(app)
+            #     load_db(app)
+            #     update_params(app,set_camera_attr_en=True)
+            return redirect(url_for('set_config_spectrometer'))
+        else:
+            return redirect(url_for('set_config_spectrometer'))
+    return app
+
+def start_webstreaming():
+    ap = argparse.ArgumentParser()
+
+    ap.add_argument("-i", "--ip", type=str, required=True,
+        help="ip address of the device")
+
+    ap.add_argument("-o", "--port", type=int, required=True,
+        help="ephemeral port number of the server (1024 to 65535)")
+
+    ap.add_argument("-loc", "--location", type=str, required=True,
+        help="ephemeral port number of the server (1024 to 65535)")
```

Comment:
wrong help

Reply:
You are right. That was a fast copy&paste and i forgot to change the description.

Reply:
I changed this help, so it's now OK